doc/source/whatsnew/v0.23.5.txt

```diff
@@ -23,6 +23,9 @@ Fixed Regressions
 - Constructing a DataFrame with an index argument that wasn't already an
   instance of :class:`~pandas.core.Index` was broken in `4efb39f
   <https://github.com/pandas-dev/pandas/commit/4efb39f01f5880122fa38d91e12d217ef70fad9e>`_ (:issue:`22227`).
+- Calling :meth:`DataFrameGroupBy.rank` and :meth:`SeriesGroupBy.rank` with empty groups
+  and ``pct=True`` was broken in `c1068d9
```

Comment:
Do we typically reference commits where regressions occurred in whatsnew notes? Would think it better to just call out the `ZeroDivisionError` instead of the commit

Reply:
Just following what was done above.

Reply:
Gotcha. Ultimately indifferent on the commit reference though I think calling out the `ZeroDivisionError` is much more useful of an indicator when either googling or looking at the whatsnew to see what has actually changed (rather than clicking through to issue or commit)

Reply:
That's fair.  I'll let the CI run its course and then make the addition.